modules/configuring-an-authentication-provider-in-central.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * cli/getting-started-cli.adoc
+:_content-type: PROCEDURE
+[id="configuring-an-authentication-provider-in-central_{context}"]
+= Configuring an authentication provider in Central
+
+[role="_abstract"]
+You can configure an authentication provider in Central and initiate the login process with the `roxctl` CLI.
+Set the variable `ROX _ENDPOINT`, initiate the login process with the `roxctl central login` command, select the authentication provider in a browser window, and retrieve the token information from the `roxctl` CLI.
```

Comment:
s/variable `ROX _ENDPOINT`/`ROX _ENDPOINT` variable 

Maybe put an _as described in the following procedure_ to indicate that all of these steps are covered in the procedure (assuming they are).